docs/src/app/components/pages/components/FontIcon/README.md

```diff
@@ -4,7 +4,7 @@ This component will render any [icon](https://www.google.com/design/spec/style/i
 defined in any stylesheet included in your project. You can use sites like [IcoMoon](https://icomoon.io/) 
 to generate custom font files, or use prebuilt fonts such as [Material icons](https://design.google.com/icons/) or 
 [Font Awesome](http://fontawesome.io/) either included in your project, or served from a public 
-[Content Delivery Network](https://en.wikipedia.org/wiki/Content_delivery_network). 
+[Content Delivery Network](https://en.wikipedia.org/wiki/Content_delivery_network). Refer to this [article](http://google.github.io/material-design-icons/#icon-font-for-the-web) to learn how to use font icons from any of the above websites in your project.
```

Comment:
Might have been copied and pasted. Will be careful about it. It does not change the look though as that line was added back.